### PR TITLE
US 1583733 Add missing language IDs - 40

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "console" for commands to F# Interactive.
- Used "console" for program output displayed on the console.

Contributes to #2192

### CODE REVIEW

**Comment by mairaw** [comment]
@cartermp do you mind validating those are the right identifiers for the F# content?

File: docs/fsharp/language-reference/attributes.md [hunk 1]
```diff
@@ -21,9 +21,7 @@ The *attribute-name* refers to the name (possibly qualified with namespaces) of
 
 The *arguments* are the arguments to the constructor for the attribute type. If an attribute has a default constructor, the argument list and parentheses can be omitted. Attributes support both positional arguments and named arguments. *Positional arguments* are arguments that are used in the order in which they appear. Named arguments can be used if the attribute has public properties. You can set these by using the following syntax in the argument list.
 
-```
 *property-name* = *property-value*
```
Comment (cartermp) [inline docs/fsharp/language-reference/attributes.md]:
```suggestion
property-name = property-value
```

File: docs/fsharp/language-reference/attributes.md [hunk 2]
```diff
@@ -21,9 +21,7 @@ The *attribute-name* refers to the name (possibly qualified with namespaces) of
 
 The *arguments* are the arguments to the constructor for the attribute type. If an attribute has a default constructor, the argument list and parentheses can be omitted. Attributes support both positional arguments and named arguments. *Positional arguments* are arguments that are used in the order in which they appear. Named arguments can be used if the attribute has public properties. You can set these by using the following syntax in the argument list.
 
-```
```
Comment (cartermp) [inline docs/fsharp/language-reference/attributes.md]: Please add as `fsharp` in conjunction with suggested change

File: docs/fsharp/language-reference/attributes.md [hunk 3]
```diff
@@ -21,9 +21,7 @@ The *attribute-name* refers to the name (possibly qualified with namespaces) of
...
 *property-name* = *property-value*
-```
```
Comment (cartermp) [inline docs/fsharp/language-reference/attributes.md]: Please add back in conjunction with suggested change

**Review by cartermp — CHANGES_REQUESTED** [review]
Small suggested change

**Comment by v-thpra** [comment]
@cartermp Made the change. Thanks! that's a good solution.

**Review by cartermp — APPROVED** [review]

**Pull request merged**